add-ons/istio/README.adoc

```diff
@@ -0,0 +1,52 @@
+:linkattrs:
+
+= Istio Add-on
+
+An addon that will deploy https://Istio.io/[istio] service mesh on to Openshift.
+
+Verify you have installed these addons, by following the link:../../README.adoc#download-and-use-community-add-ons[general readme]
+
+== minishift profile setup
+
+As https://Istio.io/[istio] setup spins up multiple pods its ideal to have a setup with the following configuration
```

Comment:
istio with capital `I`

I think you want to separate this into two lines.
```
As Istio setup spins up multiple pods.
Hence, its ideal to have a setup with the following configurations:
```

Reply:
rephrased this way

Reply:
@kameshsampath , please use `Minishift` here. `minishift` should be used in cli context.